src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+// ACTIONS
+const STATUS = 'my-app/widgets/STATUS';
+
+export default function categoriesReducer(state = {}, action = {}) {
+  switch (action.type) {
+    case STATUS:
+      return [
+        {
+          status: 'Under construction',
+        },
+      ];
```

Comment:
- [ ] The requirements state that you should return a string `"Under Construction"` when the `STATUS` is the action type. Kindly implement it according to the requirements. 👍 

![image](https://user-images.githubusercontent.com/77106962/207161591-5a78fc45-c766-4d2d-a53f-8c084c9099da.png)